src/mxarray.jl

```diff
@@ -636,16 +636,16 @@ function jsparse(mx::MxArray)
 end
 
 
-function jstring(mx::MxArray)
+function String(mx::MxArray)
```

Comment:
Actually I haven't deprecated `jstring` and `jdict`  I added    `jstring(mx::MxArray) = String(mx)` and similar to the bottom of the file. Later I can deprecate.